ur_robot_driver/src/ros/hardware_interface.cpp

```diff
@@ -428,7 +428,7 @@ void HardwareInterface::readBitsetData(const std::unique_ptr<rtde_interface::Dat
   }
 }
 
-void HardwareInterface::read(const ros::Time& time, const ros::Duration& period)
+void HardwareInterface::read(const ros::Time& time, const ros::Duration& period, bool trigger_low_frequency_logging)
```

Comment:
suggest: `publishing` instead of `logging`.

Reply:
Reverted read api change since its inherited. Use extra variable now.